### PR TITLE
Load more button

### PR DESCRIPTION
**Pull request by carlosm4247**
Description: 
Added the concept of pages to my request to create the load more mechanism. It starts on the first page and its limit is the total pages which is gotten from the API Request. When the load more button is clicked, I add the next page of results below the current articles until you get to the last page. Added some styling to create more width for the article view. Also added a filter on the articles to only show ones that don't have the same name so user isn't spammed with the same articles.

Milestones:
- User can follow candidates/races and get news on them in a feed

Test Details:
https://github.com/carlosm4247/capstone-project/assets/126618515/4cf67c88-9e93-4bfb-b1bb-0508e08da73a

### CODE REVIEW

File: capstone-ui/src/Components/Feed/Feed.jsx [hunk 1]
```diff
@@ -38,13 +48,22 @@ export default function Feed ( { loggedin }) {
       }
     }, [loggedin, user.stances, user.following]);
 
+    useEffect(() => {
+      setCurrentPage(1);
+      fetchData(1);
+    }, [searchQuery]);
+
+    useEffect(() => {
+      console.log("Total Pages:", totalPages);
```
Comment (nick-r-117) [inline capstone-ui/src/Components/Feed/Feed.jsx:57]: Please remove debug log

File: capstone-ui/src/Components/Feed/Feed.jsx [hunk 2]
```diff
@@ -38,13 +48,22 @@ export default function Feed ( { loggedin }) {
...
+    }, [totalPages]);
```
Comment (nick-r-117) [inline capstone-ui/src/Components/Feed/Feed.jsx:58]: Why do you need this useEffect?

File: capstone-ui/src/Components/Feed/Feed.jsx [hunk 3]
```diff
@@ -38,13 +48,22 @@ export default function Feed ( { loggedin }) {
       }
     }, [loggedin, user.stances, user.following]);
 
+    useEffect(() => {
+      setCurrentPage(1);
+      fetchData(1);
```
Comment (nick-r-117) [inline capstone-ui/src/Components/Feed/Feed.jsx:53]: Feels a bit redundant to reset the state to value of 1. Is it possible to combine this useEffect with the one below that handles fetching on currentPage change?

**Review by nick-r-117 — APPROVED** [review]
Just some minor comments: remove debug log statement, and useEffect usage.

File: capstone-ui/src/Components/Feed/Feed.jsx (hunk 2 above)
Comment (carlosm4247) [reply capstone-ui/src/Components/Feed/Feed.jsx:58]: This was just to check the pages with the log, I'll remove it.

**Pull request merged**